### DISCUSSION TITLE
Big-picture revising

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#129] [General]
I just discovered a manubot feature (something @pedrohbraga probably told us about, but I forgot) that will be helpful for the "big-picture" revision stage.  On the [html version](https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html) of the manuscript there is a button in the upper right corner for hypothes.is(https://hypothes.is/) integration that lets you highlight and add comments publicly.  I've added a few comments just to test it out.  I don't know what is going to happen to those comments when manubot re-builds the manuscript though...

**Comment by wavingtowaves** [comment]
Oh this is great! I think during our Feb meeting we were wondering the best
way to provide these types of revisions. Thanks for highlighting it! We'll
see what happens when manubot rebuilds the doc.

On Thu, Mar 10, 2022 at 11:33 AM Eric R Scott ***@***.***>
wrote:

> I just discovered a manubot feature (something @pedrohbraga
> <https://github.com/pedrohbraga> probably told us about, but I forgot)
> that will be helpful for the "big-picture" revision stage. On the html
> version
> <https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html>
> of the manuscript there is a button in the upper right corner for
> hypothes.is(https://hypothes.is/) integration that lets you highlight and
> add comments publicly. I've added a few comments just to test it out. I
> don't know what is going to happen to those comments when manubot re-builds
> the manuscript though...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SORTEE-Github-Hackathon/manuscript/discussions/129>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTX7SLO4BFHEOJYL45DDW3U7JE7ZANCNFSM5QNTBODA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Rob Crystal-Ornelas, PhD
Postdoctoral Scholar
Lawrence Berkeley National Lab | ESS-DIVE
Pronouns: he/him/his
